src/dotnet/Chat.UI.Blazor/Components/AccountModal/AddToContactsButton.razor

```diff
@@ -34,10 +34,12 @@
         };
 
     protected override async Task<Model> ComputeState(CancellationToken cancellationToken) {
-        var peerChatId = await AuthorUI.GetPeerChatId(Author.Id, cancellationToken);
+        var session = Session;
+        var authorId = Author.Id;
+        var peerChatId = await AuthorUI.GetPeerChatId(authorId, cancellationToken);
         if (peerChatId == PeerChatId.None)
             return new ();
-        var contact = await Contacts.GetForChat(Session, peerChatId, cancellationToken);
+        var contact = await Contacts.GetForChat(session, peerChatId, cancellationToken);
```

Comment:
shouldn't it come with ConfigureAwait(false)? otherwise not clear why you copied properties to local vars